erts/preloaded/src/erlang.erl

```diff
@@ -5219,6 +5221,64 @@ Example:
 processes() ->
     erlang:nif_error(undefined).
 
+%% The process iterator is a 2-tuple, consisting of an index to the process
+%% table and a list of process identifiers that existed when the last scan of
+%% the process table took place. The index is the starting place for the next
+%% scan of the process table.
+-opaque processes_iter_ref() :: {integer(), [pid()]}.
+
+%% processes_first/0
+-doc """
+Returns a processes iterator that can be used in
+[`processes_next/1`](`processes_next/1`).
+""".
+-doc #{ group => processes }.
+-spec processes_first() -> processes_iter_ref().
+processes_first() ->
```

Comment:
I think this name should be `processes_iterator/0` or something similar, since it does not give you the first pid, only an iterator. Also compare with for example maps:iterator() and maps:next().